spec/lib/jobs/install-os-job-spec.js

```diff
@@ -231,9 +232,13 @@ describe('Install OS Job', function () {
             return expect(job._convertInstallDisk()).to.be.rejectedWith(Error);
         });
 
-        it('should do nothing if installDisk is not specified', function() {
+        it('should set SATADOM wwid as default if installDisk is not specified', function() {
             job.options.installDisk = null;
-            return expect(job._convertInstallDisk()).to.not.be.rejected;
+            return Promise.resolve().then(function() {
```

Comment:
Since now _convertInstallDisk always return promise, we can remove this line in point of simplicity.